scripts/start-eigenda-proxy.sh

```diff
@@ -12,12 +12,15 @@ docker run -d --name eigenda-proxy-nitro-test-instance \
   -p 4242:6666 \
   -e EIGENDA_PROXY_ADDR=0.0.0.0 \
   -e EIGENDA_PROXY_PORT=6666 \
+  -e EIGENDA_PROXY_STORAGE_BACKENDS_TO_ENABLE=V1 \
+  -e EIGENDA_PROXY_STORAGE_DISPERSAL_BACKEND=V1 \
+  -e EIGENDA_PROXY_APIS_TO_ENABLE=arb \
```

Comment:
this is the wrong field, we should be providing `standard` here since its for V1: https://github.com/Layr-Labs/eigenda/tree/master/api/proxy#standard-commitment-mode